src/it/java/com/google/checkstyle/test/chapter3filestructure/toolongpackagetotestcoveragegooglesjavastylerule/PackageStatementTest.java

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code and other text files for adherence to a set of rules.
+// Copyright (C) 2001-2024 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+package com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.checkstyle.test.base.AbstractGoogleModuleTestSupport;
+
+public class PackageStatementTest extends AbstractGoogleModuleTestSupport {
+
+    @Override
+    protected String getPackageLocation() {
+        return "com/google/checkstyle/test/chapter3filestructure/"
+                + "toolongpackagetotestcoveragegooglesjavastylerule";
+    }
+
+    @Test
+    public void testPackageStatement() throws Exception {
+        final String filePath = getPath("InputPackageStatement.java");
+
+        final String[] listOfModules = {
+            "LineLength",
+            "NoLineWrap",
```

Comment:
when you will try to use full config ?

we can migrate to chapter based and then to all (in two steps). I just curious what is more preferable by you.

Reply:
Yes, send separate PR to make it be supported.
Meanwhile we can convert warn comments to violation comments, it is unavoidable work.

Reply:
new PR for testing against whole config: https://github.com/checkstyle/checkstyle/pull/15028